<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceSelectorClass</class>
 <widget class="QDialog" name="DeviceSelectorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Device Selector</string>
  </property>
  <property name="windowIcon">
   <iconset resource="DeviceSelector.qrc">
    <normaloff>:/icons/preferences-system.ico</normaloff>:/icons/preferences-system.ico</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="troubleshootingGroupBox">
     <property name="title">
      <string>Troubleshooting options (only use in case of problems)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="troubleshootingGridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="3">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Please select exactly one activated device by clicking on its name</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="1">
           <widget class="QCheckBox" name="installPostMixCheckBox">
            <property name="text">
             <string>Install APO</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="useOriginalAPOPreMixCheckBox">
            <property name="text">
             <string>Use original APO</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="preMixLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Pre-mix:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="installPreMixCheckBox">
            <property name="text">
             <string>Install APO</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5" rowspan="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="postMixLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Post-mix:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QComboBox" name="installModeComboBox">
            <item>
             <property name="text">
              <string>Install as LFX/GFX</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Install as SFX/MFX</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Install as SFX/EFX</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="useOriginalAPOPostMixCheckBox">
            <property name="text">
             <string>Use original APO</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="autoCheckBox">
            <property name="toolTip">
             <string>Automatically adjust install mode if problems with current mode are detected</string>
            </property>
            <property name="text">
             <string>Auto:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" colspan="2">
           <widget class="QCheckBox" name="allowSilentBufferCheckBox">
            <property name="toolTip">
             <string>Allows to change silent input buffers to audible output buffers, so that delayed audio can be played.

CAUTION: Some sound card drivers can produce unwanted sounds when this is enabled.</string>
            </property>
            <property name="text">
             <string>Allow silent buffer modification</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="deviceTreeWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <column>
      <property name="text">
       <string>Connector</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Device</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="requestLabel">
     <property name="text">
      <string>Please select the devices for which Equalizer APO is to be installed:</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="copyDeviceCommandAction">
   <property name="text">
    <string>Copy Device command to clipboard</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DeviceSelector.qrc"/>
 </resources>
 <connections/>
</ui>
